compiler/modules/front/codeGen.bal

```diff
@@ -1216,6 +1227,98 @@ function codeGenExpr(CodeGenContext cx, bir:BasicBlock bb, Environment env, s:Ex
     panic err:impossible("unrecognized expression type in code gen: " +  s:exprToString(expr));
 }
 
+type LiftResult record {|
```

Comment:
The names here are not consistent. You are sometimes using `NilLift`, sometimes just `Lift` and sometimes just `Nil`.

Reply:
fixed